lib/const.rb

```diff
@@ -75,20 +62,63 @@
   CREW_BUILD = 'x86_64-cros-linux-gnu'
 end
 
-CREW_COMMON_FLAGS = "'-Os -pipe -flto=auto -fuse-ld=gold'"
-CREW_COMMON_FNO_LTO_FLAGS = "'-Os -pipe -fno-lto -fuse-ld=gold'"
+CREW_COMMON_FLAGS = "'-w -Os -pipe -flto=auto -fuse-ld=gold'"
```

Comment:
Maybe instead of doing -w, doing `-Wno-error -Wno-dev` would be helpful. When I'm building packages I do use some of those package warnings to figure out why a package isn't building. -Wno-error prevents the compiler from fatally failing and -Wno-dev hides warnings meant for upstream developers.

Reply:
It would definitely be good to have `-Wno-dev` as a minimum part of CMAKE variables. I think ARCH already does this.

Reply:
Change made